src/MSBuild/TerminalLogger/TerminalLogger.cs

```diff
@@ -320,6 +325,12 @@ private void ApplyParameter(string parameterName, string? parameterValue)
             case "SHOWCOMMANDLINE":
                 TryApplyShowCommandLineParameter(parameterValue);
                 break;
+            case "SUMMARY":
+                _showSummary = true;
+                break;
+            case "NOSUMMARY":
+                _showSummary = false;
+                break;
```

Comment:
Not required for this PR but how are we going to represent the "auto" mode?

Reply:
The variable is nullable, so when we know how we want to implement the heuristics, it will be used when the user does not define an explicit parameter.